docs/linting/README.md

```diff
@@ -35,6 +38,12 @@ module.exports = {
 
 This is about the smallest config file we recommend. There's a lot more you can add to this as you further onboard, but this will be enough to get you started.
 
+:::info
+
+The `.cjs` extension will explicitly set the file to a [CommonJS module](https://nodejs.org/dist/latest-v18.x/docs/api/modules.html), in case your project has `"type": "module"` in its package.json.
+
```

Comment:
Can you add a link to https://eslint.org/docs/user-guide/configuring/configuration-files for more info? Maybe something like...

```suggestion
If your project doesn't use ESM, you can use name the file `.eslintrc.js` file instead. See [ESLint's Configuration Files docs](https://eslint.org/docs/user-guide/configuring/configuration-files) for more info.

```